installer/templates/phx_new/lib/app_name/web.ex

```diff
@@ -0,0 +1,76 @@
+defmodule <%= application_module %>.Web do
+  @moduledoc """
+  A module that keeps using definitions for controllers,
+  views and so on.
```

Comment:
The trouble with use MyApp.Web, :context is that a context belongs outside
of Web. We could have a vanilla module that you would "use" but I would
instead do my best to not have a shared "use" at first.



-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*